cli/src/semgrep/core_runner.py

```diff
@@ -435,7 +447,7 @@ def execute(self) -> int:
         with Progress(
             BarColumn(),
             MofNCompleteColumn(),
-            TextColumn("tasks"),
+            TextColumn("files"),
```

Comment:
Wanna make sure you're aware why this was called tasks instead of files.

```terminal
$ echo 1 > one.py
$ semgrep --config=auto one.py
Scanning across multiple languages:
    <multilang> |  50 rules × 2 files
         python | 355 rules × 1 file

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0/3 tasks -:--:--
```

The progress bar says there are '3 tasks' total now and I guess it would say '3 files' after your PR. I think it would be very confusing if Semgrep claimed it's scanning 3 files when you explicitly passed only one.

*Note: I see there's a bug now where we say 2 multilang files because it seems we add together the number of files for regex and for generic rules.*

Reply:
ooh, that makes sense. this work was based off of what @dlukeomalley saw in a dry run, where "tasks" was somewhat confusing. but it looks like there's a reason for it after all.

@dlukeomalley what do you think? is "tasks" still sufficient, or would a more-specific term like "targets" (which hopefully is understood not to correlate to files 1-1) be better?

Reply:
Does it need to say anything and/or show a concrete numerator/denominator? What if instead if was a percentage?

Reply:
@dlukeomalley Yes, it can be a percentage! See the revised PR.